### DISCUSSION TITLE
Uncertainty breakdown in L2 HDF files

### DISCUSSION THREAD

**Discussion by kristalikas** [nasa/HyperInSPACE#255] [Ideas]
Uncertainty Breakdown Plots is a great addition in the HyperCP! Despite it already offers to see the contributions graphically, it would be very useful if breakdowns could be exported to L2 HDF files. This would allow to analyze the uncertainty contributions due to various corrections on the measured data.

**Comment by oceancolorcoder** [comment]
@kristalikas, we have discussed this as a group recently and agreed this was something we wanted to do. It slipped in favor of higher priority fixes. Thanks for putting it here to remind us.

**Reply by kristalikas** [reply]
Great! Thank you!
